tests/query_scheduling/test_dag_collection.py

```diff
@@ -3,13 +3,18 @@
 import os
 from pathlib import Path
 import pytest
+from unittest import mock
 
 from bigquery_etl.query_scheduling.dag_collection import DagCollection
 from bigquery_etl.query_scheduling.dag import InvalidDag, DagParseException
 from bigquery_etl.query_scheduling.task import Task
 from bigquery_etl.metadata.parse_metadata import Metadata
+from bigquery_etl.dryrun import DryRun
 
 TEST_DIR = Path(__file__).parent.parent
+TEST_DRY_RUN_URL = (
+    "https://us-central1-bigquery-etl-integration-test.cloudfunctions.net/dryrun"
+)
```

Comment:
I created a dry run function in the `bigquery-etl-integration-test` which is used for the tests here. It's probably also useful to have for testing if changes are made to the function itself.

Reply:
How is that function managed? It seems potentially problematic that the function deployed in the test project could be outdated compared to the shared-prod version.

Reply:
Currently, it's been created manually in `bigquery-etl-integration-test`. But maybe there is a way to deploy the Cloud Function in https://github.com/scholtzan/cloudops-infra/tree/master/projects/data-shared/tf/modules/cloudfunctions/src/bigquery_etl_dryrun to both projects?

Reply:
I think it's sufficient for now to put a comment in the code here about how this is currently deployed manually and could get out of date with what's deployed in shared-prod.